tsl/src/fdw/data_node_scan_plan.c

```diff
@@ -466,6 +466,9 @@ data_node_scan_add_node_paths(PlannerInfo *root, RelOptInfo *hyper_rel)
 	 * Create estimates and paths for each data node rel based on data node chunk
 	 * assignments.
 	 */
+#if PG15_GE
+	hyper_rel->live_parts = NULL;
```

Comment:
It might be cleaner to declare a new Bitmapset here, then add all the rels, and set the new bitmapset in hyper_rel after the loop where we "reset" the other partitioning info:

```
hyper_rel->part_rels = data_node_rels;
hyper_rel->nparts = ndata_node_rels;
hyper_rel->live_rels = data_node_live_rels;
```

```suggestion
	Bitmapset *data_node_live_rels = NULL;
```